tests/shared/src/test/scala/cats/tests/ParallelSuite.scala

```diff
@@ -390,6 +390,37 @@ class ParallelSuite
     }
   }
 
+  test("ParFlatMapN over List should be consistent with parMapN flatten") {
+    forAll { (as: List[Int], bs: List[Int], cs: List[Int]) =>
+      val mf: (Int, Int, Int) => List[Int] = _ :: _ :: _ :: Nil
```

Comment:
Sorry for bikeshedding and nitpicking :)
Here and in some test cases below – does `mf` have to be just a concatenation?
Can it be generated by Scalacheck instead along with the other inputs?

Reply:
> Sorry for bikeshedding and nitpicking :) 

No worries, cats is a reliable library, so, naturally, it has to be tight about compatibility, performance, correctness, consistency, etc.

> Here and in some test cases below – does `mf` have to be just a concatenation? Can it be generated by Scalacheck instead, along with the other inputs?

You're right; I used scalacheck to generate the functions. It's just too mindblowing for me that it can even generate functions that it's not natural for me to use it :D